Detectors/ITSMFT/ITS/tracking/src/IOUtils.cxx

```diff
@@ -141,53 +141,53 @@ int ioutils::loadROFrameData(const o2::itsmft::ROFRecord& rof, ROframe& event, g
-{
-  std::vector<std::unordered_map<int, Label>> labelsMap{};
-  std::unordered_map<int, Label> currentEventLabelsMap{};
-  std::ifstream inputStream{};
-  std::string line{};
-  int monteCarloId{}, pdgCode{}, numberOfClusters{};
-  float transverseMomentum{}, phi{}, pseudorapidity{};
+// std::vector<std::unordered_map<int, Label>> ioutils::loadLabels(const int eventsNum, const std::string& fileName)
+// {
+//   std::vector<std::unordered_map<int, Label>> labelsMap{};
+//   std::unordered_map<int, Label> currentEventLabelsMap{};
+//   std::ifstream inputStream{};
+//   std::string line{};
+//   int monteCarloId{}, pdgCode{}, numberOfClusters{};
+//   float transverseMomentum{}, phi{}, pseudorapidity{};
 
-  labelsMap.reserve(eventsNum);
+//   labelsMap.reserve(eventsNum);
 
-  inputStream.open(fileName);
-  std::getline(inputStream, line);
+//   inputStream.open(fileName);
+//   std::getline(inputStream, line);
 
-  while (std::getline(inputStream, line)) {
+//   while (std::getline(inputStream, line)) {
 
-    std::istringstream inputStringStream(line);
+//     std::istringstream inputStringStream(line);
 
-    if (inputStringStream >> monteCarloId) {
+//     if (inputStringStream >> monteCarloId) {
 
-      if (monteCarloId == EventLabelsSeparator) {
+//       if (monteCarloId == EventLabelsSeparator) {
 
-        labelsMap.emplace_back(currentEventLabelsMap);
-        currentEventLabelsMap.clear();
+//         labelsMap.emplace_back(currentEventLabelsMap);
+//         currentEventLabelsMap.clear();
 
-      } else {
+//       } else {
 
-        if (inputStringStream >> transverseMomentum >> phi >> pseudorapidity >> pdgCode >> numberOfClusters) {
+//         if (inputStringStream >> transverseMomentum >> phi >> pseudorapidity >> pdgCode >> numberOfClusters) {
 
-          if (std::abs(pdgCode) == constants::pdgcodes::PionCode && numberOfClusters == 7) {
+//           if (std::abs(pdgCode) == constants::pdgcodes::PionCode && numberOfClusters == 7) {
 
-            currentEventLabelsMap.emplace(std::piecewise_construct, std::forward_as_tuple(monteCarloId),
-                                          std::forward_as_tuple(monteCarloId, transverseMomentum, phi,
-                                                                pseudorapidity, pdgCode, numberOfClusters));
-          }
-        }
-      }
-    }
-  }
+//             currentEventLabelsMap.emplace(std::piecewise_construct, std::forward_as_tuple(monteCarloId),
+//                                           std::forward_as_tuple(monteCarloId, transverseMomentum, phi,
+//                                                                 pseudorapidity, pdgCode, numberOfClusters));
+//           }
+//         }
+//       }
+//     }
+//   }
 
-  labelsMap.emplace_back(currentEventLabelsMap);
+//   labelsMap.emplace_back(currentEventLabelsMap);
 
-  return labelsMap;
-}
+//   return labelsMap;
+// }
```

Comment:
I guess this method is obsolete and can just be removed, if you don't have other plans.